### PR TITLE
[battery_manager] Clarify Permission Labels

### LINKED ISSUE #6683: [battery_manager] Edit permission
**Describe the bug**
The edit permission description on the front end is ambiguous in the permission list in user accounts. ‘Add, activate, and deactivate entries in Test Battery’: this description should be changed (confusing for user support) or test plan should be more precise. Still can’t access the module even with the 2 permissions 

**To Reproduce**
Steps to reproduce the behavior (attach screenshots if applicable):
1. Go to user accounts
2. In the permission list, you should see the permission ‘Add, activate, and deactivate entries in Test Battery’. I had to go to my back end to make sure that was the right permission
3. Still can't access the module even with the 2 (view and edit) permission for this module

### PR DESCRIPTION
**Pull request by HenriRabalais**
#### Testing instructions (if applicable)

1. Run to patch.
2. Go to User accounts and make sure that the Permission labels match the labels in the patch.

#### Link(s) to related issue(s)

* Resolves #6683

### CODE REVIEW

**Comment by cmadjar** [comment]
@pierre-p-s can you test and review this PR?

**Comment by pierre-p-s** [comment]
> @pierre-p-s can you test and review this PR?

@cmadjar yes Ill test it now

File: modules/battery_manager/test/TestPlan.md [hunk 1]
```diff
@@ -31,34 +31,34 @@ Test Battery.
...
-  5. Clicking on Add tab should hide the data table and display a form with the following fields:
+  5. Clicking on the "New Test" button should trigger a modal window displaying a form with the following fields:
      `Instrument`, `Minimum age (days)`, `Maximum age(days)`, `Stage`, `Subproject`, `Visit Label`, `Site`, `First Visit`,
      and `Instrument Order`.
   6. Clicking on Edit in **Edit Metadata** takes you to a new page with a form with the following fields:
      `Instrument`, `Minimum age (days)`, `Maximum age(days)`, `Stage`, `Subproject`, `Visit Label`, `Site`, `First Visit`,
      and `Instrument Order`.
 
-### Add tab
+### Add Button
```
Comment (pierre-p-s) [inline modules/battery_manager/test/TestPlan.md:44]: Instead of 'Add Button', I think it should be 'New Test Button'

**Comment by pierre-p-s** [comment]
It worked as intended when I tested the permissions. 
Except the little comment above the code review was fine.

**Comment by HenriRabalais** [comment]
@pierre-p-s fixed!

**Pull request merged**